### ISSUE TITLE
Featured Content in Business or Portfolio

### ISSUE THREAD

**Issue by sixhours** [Automattic/theme-pattern-library#68]
It's worth researching (if you haven't already) to see how common the use of Featured Content is within these two types of themes. I expected to see Featured Content pop up when creating a Business or Portfolio theme for some reason, and was surprised when they didn't.

**Comment by karmatosed** [comment]
Absolutely, I can do more research. I did feel Business was borderline, so it's good you bring it up. It was close to 50/50 on Business having featured content or using the page as the featured content. I think we could look at bolstering the front page template for business - maybe this is part of that.

Portfolio, I'm torn about it. Perhaps this needs to be considered as an option for portfolio in that case. With v2 the plan will be once type is selected you can drill down into more options. This makes sense to bring in then to me. However, if we all expect it and research backs it up, absolutely we should bring in.
